docker-compose.yml

```diff
@@ -2,11 +2,10 @@ services:
   app:
     build: .
     command:
-      - sh
+      - bash
```

Comment:
Did plain old `sh` not work?

Reply:
Not a strictly necessary change in this PR. I found that bash seems to be better at forwarding interrupt signals to child processes so services shutdown more quickly and cleanly.